### DISCUSSION TITLE
Syntax for transloco keys?

### DISCUSSION THREAD

**Discussion by konstantinschuette** [ngneat/transloco#590] [General]
Hey,

I haven't found the syntax for the keys. Should they be in camel case?

**Comment by shaharkazaz** [comment]
@konstantinschuette I'm not sure I understand your question, can you please elaborate?
Generally speaking, there is no syntax for the keys.
Take a look at [this example](https://codesandbox.io/s/ngneat-transloco-kn52hs).

**Reply by konstantinschuette** [reply]
Some colleagues created keys with full sentences like `This is a test of hello world.`
That causes the problem, that you cannot find that key with the `transloco-keys-manager`.
So I think it would be good to recommend a syntax, in order to take full advantage of all transloco plugins.

**Reply by shaharkazaz** [reply]
@konstantinschuette That doesn't sound right, I just ran the command with this key in a local test env and it worked.
Please share a reproduction of the scenario.

**Reply by stones** [reply]
I am not sure if this the same issue, but if I have kebab case scope, the scope resolver converts it to camelCase:
```
     if (isScopeObject(provider)) {
        const { scope, alias = toCamelCase(scope) } = provider as ProviderScope;
        this.translocoService._setScopeAlias(scope, alias);

        return scope;
      }
```
